### PR TITLE
Added EventBridge rules scripts

### PR DESCRIPTION
**Pull request by jim-sheldon**
We can use these two new scripts in combination with aws.py to create new EventBridge rules

### CODE REVIEW

File: ingestion/functions/make_rules.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        if not target_name:
+            print(f"Missing target name in rule {rule}")
+            sys.exit(1)
+        if not source_name:
+            print(f"Missing source name in rule {rule}")
+            sys.exit(1)
+        if not job_name:
+            print(f"Missing job name in rule {rule}")
+            sys.exit(1)
+        cmd = f"./aws.py schedule {rule_name} -t {target_name} -s \"{source_name}\" -j {job_name}"
```
Comment (abhidg) [inline ingestion/functions/make_rules.py:35]: I would try to import and call out, but aws.py isn't implemented that way, so not a blocker

**Review by abhidg — CHANGES_REQUESTED** [review]
Looks good other than a few small changes!

File: ingestion/functions/aws.py [hunk 2]
```diff
@@ -155,12 +156,31 @@ def register(self):
                     )
                 )
             )
+            if args.parser == parser:
```
[the remark this answers is not in the document]
Comment (jim-sheldon) [reply ingestion/functions/aws.py]: My bad, I'd meant to rename the variable coming back from `get_parser_name_source`, not override the `ArgumentParser`. Fixed.

File: ingestion/functions/define_rules.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+import boto3
+
+import common.common_lib as common_lib
+
+
+DEV = "dev"
+PROD = "prod"
+
+DEV_CURATOR_URL = "https://dev-curator.ghdsi.org/api/sources"
```
[the remark this answers is not in the document]
Comment (jim-sheldon) [reply ingestion/functions/define_rules.py]: Updated.

File: ingestion/functions/define_rules.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+sources = response.json().get("sources")
+
+rules = []
+
+for source in sources:
+	source_name = source.get("name")
+	source_id = source.get("_id")
+	source_rule = SOURCE_RULE.copy()
+	source_rule["rule_name"] = f"{source_id}-{env}"
+	source_rule["target_name"] = f"{source_name.lower().replace(' ', '_')}_ingestor"
```
[the remark this answers is not in the document]
Comment (jim-sheldon) [reply ingestion/functions/define_rules.py]: Agreed, updated and will use this library going forward.

File: ingestion/functions/make_rules.py (hunk 1 above)
Comment (jim-sheldon) [reply ingestion/functions/make_rules.py:35]: Maybe better down the road to put `define_rules.py` and `make_rules.py` functions inside `aws.py`?
Going to leave it alone for now.

Comment (abhidg) [reply ingestion/functions/make_rules.py:35]: Yes, good idea for strongcat.

**Review by abhidg — APPROVED** [review]
LGTM

**Pull request closed without merging**